drools-parser/src/main/antlr4/org/drools/parser/DRLLexer.g4

```diff
@@ -124,7 +124,7 @@ DRL_BIG_INTEGER_LITERAL
 /////////////////
 
 HASH : '#';
-UNIFY :	':=' ;
+DRL_UNIFY :	':=' ;
```

Comment:
THanks @tkobayas 
Out of curiosity: why an "equal" operator is called "UNIFY" ?

Reply:
Hi @gitgabrio 

`:=` works like a composite of a bind variable decalaration and `==`. Basically, The same name variables cannot be declared, but you can declare the same name variables by using `:=` to ensure they have the same value. So it's called `unify`. 

For example,
```
rule Hello
  when
    $person : Person($name := name)
    $pet : Pet($name := name)
  then
    System.out.println("Hello, " + $person + ", " + $pet);
end
```

The rule matches when a Person and a Pet have the same name. 

It can be re-written like
```
rule Hello
  when
    $person : Person($name : name)
    $pet : Pet(name == $name)
  then
    System.out.println("Hello, " + $person + ", " + $pet);
end
```